core/src/apps/webauthn/fido_seed.py

```
@@ -0,0 +1,18 @@
+def ensure_fido_seed(func):
```

Ruff (0.8.2)

Missing return type annotation for public function `ensure_fido_seed`

(ANN201)